.github/workflows/delete-old-artifacts.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Delete old artifacts'
+on:
+  schedule:
+    - cron: '0 15 * * *' # every day
```

Comment:
As we have nightly builds and artifacts are useful for debugging deployment scripts, how about wait longer days before delete old artifacts, like a week?

Reply:
Hmm… I misunderstood the code.  It seems to already work as I said.